### PR TITLE
Fit brinch

### PR DESCRIPTION
**Pull request by dp-rice**
Fit the model to the Brinch data (as well as any changes that have been made to the pathogen files since the June all-hands memo).

Model fit:
overall-boxen

Matching reads table:
matching-reads-table

Note that this partially breaks `plot_summaries.py` but this will be revised significantly soon to improve the figures.

### CODE REVIEW

File: fit.py [hunk 1]
```diff
@@ -22,14 +29,15 @@ def start(num_samples: int, plot: bool) -> None:
...
-        _,
         pathogen_name,
+        tidy_name,
         predictor_type,
         taxids,
         predictors,
     ) in predictors_by_taxid():
         taxids_str = "_".join(str(t) for t in taxids)
         for study, bioprojects in target_bioprojects.items():
+            print(pathogen_name, study)
```
Comment (jeffkaufman) [inline fit.py:40]: is this intentional?

File: stats.py [hunk 2]
```diff
@@ -110,7 +110,16 @@ class DataPoint(Generic[P]):
     sample: Sample
     attrs: SampleAttributes
     viral_reads: int
-    predictor: P
+    predictor: P | None
+
+    def get_predictor_value(self) -> float:
+        # TODO: Handle missing data
```
Comment (jeffkaufman) [inline stats.py:116]: what's this TODO about?

**Review by jeffkaufman — APPROVED** [review]

File: fit.py (hunk 1 above)
Comment (dp-rice) [reply fit.py:40]: I was using it for debugging. Doesn't hurt anything, but it doesn't really help either. I'll remove it.

File: stats.py (hunk 2 above)
Comment (dp-rice) [reply stats.py]: If we eventually update the Stan model to allow for some of the samples to be missing the predictor data, we'll change this to return the value Stan is expecting for missing data (e.g., NaN). Comment updated

**Pull request merged**